docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -1052,6 +1052,45 @@ var.kind_of?(String)
 
 * https://rubystyle.guide#is-a-vs-kind-of
 
+== Style/ClassEqualityComparison
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| Yes
+| 0.93
+| -
+|===
+
+This cop enforces the use of `Object#instance_of?` instead of class comparison
```

Comment:
Btw, you can use `kind_of?` as well. I think we have some other cop for this, so that one should honor the config of the existing cop.

Reply:
`kind_of?` is an alias for `is_a?`, which checks also superclasses. So this should be `instance_of?` for sure.

Reply:
I keep mixing those up. :-)